hw/ip/socdbg_ctrl/data/socdbg_ctrl.hjson

```diff
@@ -0,0 +1,345 @@
+      name:    "soc_dbg_state"
+      package: "lc_ctrl_state_pkg"
+      act:     "rcv"
+    }
+    { struct:  "soc_dbg_policy"
+      type:    "uni"
+      name:    "socdbg_policy_bus"
+      package: "socdbg_ctrl_pkg"
+      act:     "req"
+    }
+    // Broadcast from LC
+    { struct:  "lc_tx"
+      type:    "uni"
+      name:    "lc_escalate_en"
+      act:     "rcv"
+      default: "lc_ctrl_pkg::Off"
+      package: "lc_ctrl_pkg"
+      desc:    '''
+               Life cycle escalation enable coming from life cycle controller.
+               This signal moves all FSMs within OTP into the error state.
+               '''
+    }
+    { struct:  "lc_tx"
+      type:    "uni"
+      name:    "lc_hw_debug_en"
+      act:     "rcv"
+      default: "lc_ctrl_pkg::Off"
+      package: "lc_ctrl_pkg"
+      desc:    '''
+               Multibit life cycle hardware debug enable signal coming from life cycle controller,
+               asserted when the hardware debug mechanisms are enabled in the system.
+               '''
+    },
+    { struct:  "lc_tx"
+      type:    "uni"
+      name:    "lc_creator_seed_sw_rw_en"
+      act:     "rcv"
+      default: "lc_ctrl_pkg::Off"
+      package: "lc_ctrl_pkg"
+      desc:    '''
+               Provision enable qualifier coming from life cycle controller.
+               This signal enables SW read / write access to the RMA_TOKEN and CREATOR_ROOT_KEY_SHARE0 and CREATOR_ROOT_KEY_SHARE1.
+               '''
+    }
+    { struct:  "lc_tx"
+      type:    "uni"
+      name:    "lc_seed_hw_rd_en"
+      act:     "rcv"
+      default: "lc_ctrl_pkg::Off"
+      package: "lc_ctrl_pkg"
+      desc:    '''
+               Seed read enable coming from life cycle controller.
+               This signal enables HW read access to the CREATOR_ROOT_KEY_SHARE0 and CREATOR_ROOT_KEY_SHARE1.
+               '''
+    }
+    { struct:  "lc_tx"
+      type:    "uni"
+      name:    "lc_dft_en"
+      act:     "rcv"
+      default: "lc_ctrl_pkg::Off"
+      package: "lc_ctrl_pkg"
+      desc:    '''
+               Test enable qualifier coming from life cycle controller.
+               This signals enables the TL-UL access port to the proprietary OTP IP.
+               '''
+    }
+    { struct:  "lc_tx"
+      type:    "uni"
+      name:    "lc_check_byp_en"
+      act:     "rcv"
+      default: "lc_ctrl_pkg::Off"
+      package: "lc_ctrl_pkg"
+      desc:    '''
+               Life cycle partition check bypass signal.
+               This signal causes the life cycle partition to bypass consistency checks during life cycle state transitions in order to prevent spurious consistency check failures.
+               '''
+    }
+  ]
+
+  // REGISTER definition
+  regwidth: "32"
+  registers: {
+    core: [
+        // CTRL register
+        { name: "DEBUG_POLICY_CTRL"
+          desc: "Debug Policy Control"
+          swaccess: "rw"
+          hwaccess: "hro"
+          fields: [
+            { bits: "3:0"
```

Comment:
Can this be made into an enum, in the same way as the life cycle state registers?